extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js

```diff
@@ -21,6 +21,15 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (w, pon
 			w.location = w.templatePageUrl;
 			return true;
 		};
+
+		/**
+		 * redirects to source editor
+		 * @returns {Boolean}
+		 */
+		this.redirectToSourceEditor = function () {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)